index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Portfolio Website</title>
+</head>
+<body >
+    <header class="header-logo">
+        <h2 class="logo">My Logo</h2>
+        <img src="images/menu-rounded.png" alt="" class="image-icon"> 
+         
+    </header>
+
+    <div class="all-body">
+        <section class="main">
+            <h2 class="title">I'm Mariane Glad to see you!</h2>
+            <p class="intro">I'm a software developer! I can help you build a product , 
+                feature or website Look through some of my work and experience! 
+                If you like what you see and have a project you need coded, don't 
+                hestiate to contact me.</p>
+        </section>
+
+        <div class="connect">
+
+            <div class="lets-connect">
+                <p>LET'S CONNECT</p>
+            </div>
+
+            <img src="images/Vector.png" class="icons">
+            <img src="images/Vector (1).png" class="icons"><img src="images/Vector (2).png" class="icons">
+            <img src="images/Vector (3).png" class="icons">
+            <img src="images/Vector (4).png" class="icons">
+            <img src="images/Vector (5).png" class="icons">
```

Comment:
- [ ] Dear @MarkYegon7, for the list of icons we are expected to use a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag). this provides sematic advantage to our document.

Reply:
When I include the ul, li, and a  tag it adds  bullets and underlines the icons contrary to the figma example